modules/gapi/test/cpu/gapi_core_tests_fluid.cpp

```diff
@@ -285,260 +285,4 @@ INSTANTIATE_TEST_CASE_P(CopyTestFluid, CopyTest,
                                        cv::Size(128, 128)),
                                 Values(-1),
                                 Values(CORE_FLUID)));
-
-//----------------------------------------------------------------------
-// FIXME: Clean-up test configurations which are enabled already
```

Comment:
@anton-potapov  Tests with the same name located in `gapi_core_test_cpu.cpp`.  I deleted them from here. Is it right way? Please review.

Reply:
@Volskig  
I think the FIXME comment here assumes double-checking each test on the case-by case basis: if the corresponding ...TestFluid case is already enabled. If you have done this case-by-case check, then I think it is OK to simply remove the redundant cases here.

Reply:
+1